cmds/core/pci/pci.go

```diff
@@ -20,25 +20,55 @@ package main
 
 import (
 	"encoding/json"
+	"errors"
+	"flag"
 	"fmt"
 	"io"
 	"log"
 	"os"
 	"strconv"
 	"strings"
 
-	flag "github.com/pborman/getopt/v2"
 	"github.com/u-root/u-root/pkg/pci"
 )
 
-var (
-	numbers   = flag.Bool('n', "Show numeric IDs")
-	devs      = flag.StringLong("select", 's', "*", "Devices to match")
-	dumpJSON  = flag.BoolLong("json", 'j', "Dump the bus in JSON")
-	verbosity = flag.Counter('v', "verbosity")
-	hexdump   = flag.Counter('x', "hexdump the config space")
-	readJSON  = flag.StringLong("JSON", 'J', "", "Read JSON in instead of /sys")
-)
+type cmd struct {
+	w         io.Writer
+	osargs    []string
+	args      []string
+	numbers   bool
+	devGlobs  string
+	dumpJSON  bool
+	verbosity int
+	hexdump   int
+	readJSON  string
+	flags     *flag.FlagSet
+}
+
+// ErrBadJSON is for any JSON Unmarshal error.
+// The JSON package predates wrapped errors. This error is a placeholder for all
+// "something went wrong" JSON parsing errors.
+var ErrBadJSON = errors.New("JSON parsing failed")
```

Comment:
nit: probably no need to export this, it's a main package.

Reply:
ok.